pkg/aws/helpers.go

```diff
@@ -1000,3 +991,63 @@ func IsHostedCPManagedPolicies(cluster *cmv1.Cluster) bool {
 func IsHostedCP(cluster *cmv1.Cluster) bool {
 	return cluster.Hypershift().Enabled()
 }
+
+func ConvertToTagPointers(tags []iamtypes.Tag) []*iamtypes.Tag {
+	tagPointers := make([]*iamtypes.Tag, len(tags))
+	for i, tag := range tags {
+		t := tag
+		tagPointers[i] = &t
+	}
+	return tagPointers
+}
+
+func buildDescribeStacksInput(stackName string) *cloudformation.DescribeStacksInput {
+	return &cloudformation.DescribeStacksInput{
+		StackName: aws.String(stackName),
+	}
+}
+
+func waitForStackCreateComplete(ctx context.Context, cfClient client.CloudFormationApiClient, stackName string) error {
+	waiter := cloudformation.NewStackCreateCompleteWaiter(cfClient)
+
+	maxWaitDur := 5 * time.Minute // Max Wait time
+
+	params := buildDescribeStacksInput(stackName)
+
+	// You can also use WaitForOutput if you need the output
+	err := waiter.Wait(ctx, params, maxWaitDur, func(o *cloudformation.StackCreateCompleteWaiterOptions) {
+		// Optionally set MinDelay, MaxDelay, and other options here
+	})
+
+	return err
+}
+
+func waitForStackUpdateComplete(ctx context.Context, cfClient client.CloudFormationApiClient, stackName string) error {
+	waiter := cloudformation.NewStackUpdateCompleteWaiter(cfClient)
+
+	maxWaitDur := 5 * time.Minute // Max Wait time
```

Comment:
@den-rgb Shall we make this a `const` and then simply refer to it in the three functions that have the 5 minute wait?

Are these functions tested anywhere?

Reply:
They're tested as part of this test case: https://github.com/den-rgb/rosa/blob/OCM-3963-x/pkg/aws/client_test.go#L68

Reply:
I've also done manual testing verifying they work

Reply:
Thanks @den-rgb